pkg/controller/metrics/metrics.go

```diff
@@ -305,9 +305,6 @@ func (mc *Calculator) Start(stopCh <-chan struct{}) error {
 		}
 
 		mc.calculateSelectorSyncSetMetrics(mcLog)
-
-		elapsed := time.Since(start)
-		mcLog.WithField("elapsed", elapsed).Info("metrics calculation complete")
```

Comment:
any reason why this was dropped?

Reply:
Duplicated with another identical calculation.